### DISCUSSION TITLE
Dark Mode Standard

### DISCUSSION THREAD

**Discussion by ThorstenUm** [blakeblackshear/frigate#13120] [Ideas]
Could “Dark Mode” be set as default or saved per user? The background is that I reset the cookies every time I close the browser.

**Comment by NickM-27** [comment]
the default is system default, is your system not set to dark mode?

**Comment by ThorstenUm** [comment]
No, the system default is otherwise light.

**Comment by plopes9000** [comment]
I have frigate running on a windows machine with Edge in kiosk mode on a touchscreen. The windows machine shuts down for the night and gets automatically restarted every day - this saves a few hours of electricity. Because Edge is running in kiosk mode, no browser cache is kept, hence, every day I have to 1) set frigate to dark mode again, and 2) manually set the window size to the available space in birdeye's view. Would it be possible to make these two settings available in the config file as the defaults?  e.g. default_theme: dark, birdseye_viewport: max or similar ....
